.github/workflows/ci.yml

```diff
@@ -59,8 +59,8 @@ jobs:
       - uses: actions/checkout@v4
       - uses: DavidAnson/markdownlint-cli2-action@v18
         with:
-          config: '.markdownlint.yaml'
-          globs: '**/README.md'
+          config: ".markdownlint.yaml"
```

Comment:
So we want to use `"` instead of `'` in `ci.yml`?
I don't have a strong preference, but I can change it in other repos if yes.